cpp/src/cluster/detail/kmeans_balanced.cuh

```diff
@@ -141,6 +142,53 @@ inline std::enable_if_t<std::is_floating_point_v<MathT>> predict_core(
                         raft::compose_op<raft::cast_op<LabelT>, raft::key_op>());
       break;
     }
+    case cuvs::distance::DistanceType::CosineExpanded: {
```

Comment:
We should update the doc strings in the public API also. If the doc strings don't currently specify which metrics are supported in ivf flat then they should.